src/npg_irods/exception.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2022 Genome Research Ltd. All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# @author Keith James <kdj@sanger.ac.uk>
+
+from typing import Any
+
+
+class DataManagementError(Exception):
+    """The base class of all exceptions originating in this package."""
```

Comment:
```suggestion
    """The base class of all exceptions originating in this module."""
```
I don't know that every exception in the package (or possibly even in this module) is going to be a data management error necessarily.

Reply:
I'd like to move them if possible. Let's keep this focussed for now, by using your suggestion.